palantirnet/st-bernards
# MSSB-48: Adjust sidebar menu for mobile screens

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-48

## Description

Adjust sidebar menu for mobile and rewire layout so this can happen.

## Testing instructions

1. Visit `http://st-bernards.local/services/pediatrics/pediatric-neurology`
1. Resize with `Chrome Inspector`
1. Do you see the sidebar menu at the bottom `Also in this Section`?
1. Does it function correctly? Does it look correct?
